CHANGELOG.md

```diff
@@ -8,7 +8,20 @@ Drake is undergoing continuous development and cannot yet offer a stable, docume
 Descriptions of Pull Requests
 ------------------------------
 
-### ([@liangfok][], [#1953][]) - Replacement of valuecheckMatrix() with CompareMatrices()
+
+### ([@liangfok][], [#1985][]) - Added Max Delay and Warn command line options to quadrotor's unit test
```

Comment:
1985 need not appear.  The defaults are the same, and its unlikely to be anything users care to play with in terms of a new feature.